airflow/contrib/utils/log/task_handler_with_custom_formatter.py

```diff
@@ -37,8 +50,9 @@ def set_context(self, ti):
         if prefix:
             _, self.prefix_jinja_template = parse_template_string(prefix)
             rendered_prefix = self._render_prefix(ti)
-
-        self.setFormatter(logging.Formatter(rendered_prefix + ":" + self.formatter._fmt))
+        rendered_formatter = \
```

Comment:
This change shouldn't be needed, the maximum line length is 110 chars.

Reply:
Ah! Agreed.